x-pack/legacy/plugins/monitoring/server/telemetry_collection/get_high_level_stats.test.ts

```diff
@@ -243,9 +232,9 @@ describe('get_high_level_stats', () => {
     it('returns clusters', async () => {
       callWith.withArgs('search').returns(Promise.resolve(response));
 
-      expect(
-        await getHighLevelStats(server, callWith, clusterUuids, start, end, product)
-      ).toStrictEqual(expectedClusters);
+      expect(await getHighLevelStats(callWith, clusterUuids, start, end, product, 1)).toStrictEqual(
```

Comment:
nit: I'd prefer an explicit declaration of `maxBucketSize=1` being passed in here. Numbers tend to go 'missing'  when directly embedded in code.